security.rst

```diff
@@ -644,6 +648,16 @@ Securing other Services
 
 See :doc:`/security/securing_services`.
 
+Securing Individual Objects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Most applications require more specific access rules. For instance, a User
```

Comment:
I'd replace `User` by a `user` ... because here "user" looks like a common word, instead of the name of a Doctrine entity.